### PR TITLE
ICMSLST-681 Refactor authorise views and fix checklist validation bug.

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: web/domains/case/_import/urls.py [hunk 1]
```diff
@@ -64,12 +64,6 @@
                 ),
                 path("licence/edit", views.edit_licence, name="edit-licence"),
                 path("licence/preview/", views.preview_licence, name="preview-licence"),
-                #
```
Comment (oskusalerma) [inline web/domains/case/_import/urls.py:67]: can you change the comment on line 58-59 to say "these are..."

File: web/domains/case/urls.py [hunk 2]
```diff
@@ -150,16 +156,8 @@
...
-                # path("authorisation/", views.authorisation, name="authorisation"),
-                # path(
-                #     "start-authorisation/", views.start_authorisation, name="start-authorisation"
-                # ),
-                # path(
-                #     "cancel-authorisation/",
-                #     views.cancel_authorisation,
-                #     name="cancel-authorisation",
-                # ),
+                # Application Authorisation (import/export
```
Comment (oskusalerma) [inline web/domains/case/urls.py:159]: missing end paren

File: web/domains/case/urls.py [hunk 3]
```diff
@@ -123,6 +123,12 @@
     ),
 ]
 
+authorisation_urls = [
+    path("manage/", views.manage_authorisation, name="manage-authorisation"),
+    path("start-authorisation/", views.start_authorisation, name="start-authorisation"),
```
Comment (oskusalerma) [inline web/domains/case/urls.py:128]: these are all prefixed with authorisation, can be just start / cancel (the URLs)

**Review by kevinetienne — APPROVED** [review]

File: web/domains/case/views.py [hunk 4]
```diff
@@ -1420,3 +1422,121 @@ def _prepare_wood_quota_response(
...
+                url = reverse(
+                    "case:prepare-response",
+                    kwargs={"application_pk": application.pk, "case_type": case_type},
+                )
+                html = f"<a href='{url}'>Please complete start and end dates.</a>"
+                application_errors.append(html)
+
+        # Export only checks
+        # if case_type == "export":
+        #     ...
```
Comment (oskusalerma) [inline web/domains/case/views.py:1462]: TODO comment here, maybe even raise NotImplementedError...?

also elif

File: web/domains/case/views.py [hunk 5]
```diff
@@ -1420,3 +1422,121 @@ def _prepare_wood_quota_response(
...
+
+        # Import & export checks
+        if application.decision == model_class.REFUSE:
+            url = reverse(
```
Comment (oskusalerma) [inline web/domains/case/views.py:1466]: maybe pull this out into prepare_url, it's constructed a few times?

**Review by oskusalerma — CHANGES_REQUESTED** [review]

**Review by oskusalerma — APPROVED** [review]

**Pull request merged**